src/commands/retention-policies/assign.js

```diff
@@ -42,6 +50,31 @@ RetentionPoliciesAssignCommand.flags = {
 	'assign-to-id': flags.string({
 		description: 'The ID of the content to assign the retention policy to',
 	}),
+	'filter-field': flags.string({
+		description: 'Metadata fields to filter against, if assigning to a metadata template.' +
+			'Allow properties: field, value. Example: field=foo,value=bar',
+		multiple: true,
+		parse(input) {
+			const filter = {};
+			input.split(',').forEach((pair) => {
+				const [key, value] = pair.split('=');
+				switch (key) {
+					case 'field':
+						filter.field = value;
+						break;
+					case 'value':
+						filter.value = value;
+						break;
+					default:
```

Comment:
We can simplify one more thing.
Considering that in this case we always need to provide `field` and `value`(we don't have to check if if has specific value like in sign for instance)  we can simplify the way of providing these parameters and instead of:
`--filter-field=field=fieldName,value=fieldValue`
allow this syntax:
`--filter-field=fieldName=fieldValue`

What do you think about this? Can you apply this changes?